client/src/components/layers/EditAnnotationLayer.vue

```diff
@@ -146,6 +141,8 @@ export default {
               };
             }
             this.$emit('update:geojson', geojson);
+          } else {
+            this.$emit('update:editing', false);
```

Comment:
This breaks annotation.  

1. put track in editing state
2. Tap `f` or `d`.
3. Track editing state is removed.

Reply:
I forgot about `f` and `d`, been using `left/right` and the editing state persists.

Reply:
This has been updated with some comments and documentation to explain what is occuring.